docs/myriad-configuration.md

```diff
@@ -26,6 +26,12 @@ frameworkName: MyriadAlpha
 # Myriad's mesos framework role.
 frameworkRole: someRoleName
 
+# Myriad's mesos framework user (Defaults to user running the resource manager if absent, necessary for remote distribution).
```

Comment:
What do you mean "necessary for remote distribution"? It defaults to a user, so it's always set anyway.
And you can specify that this is the user that runs the NodeManagers.